LocationIQ Collection v2.0.0.postman_collection.json

```diff
@@ -1118,13 +1160,19 @@
 								{
 									"key": "accept-language",
 									"value": "en",
-									"description": "Preferred language for showing search results. Defaults to en. Unlike Search and Reverse endpoints, the Autocomplete endpoint supports only a single 2 digit language code. The Autocomplete endpoint currently support 4 languages: English en, French fr, German de and Italian it. If you'd like us to add support for a specific language, please reach out.",
+									"description": "Preferred language for showing search results. Defaults to en. Unlike Search and Reverse endpoints, the Autocomplete endpoint supports only a single 2 digit language code. The Autocomplete endpoint currently support 7 languages: English en, Dutch nl, French fr, German de, Italian it, Spanish es and Swedish sv.",
 									"disabled": true
 								},
 								{
 									"key": "tag",
 									"value": "place",
-									"description": "Restricts the autocomplete search results to elements of specific OSM class and type.  Example - To restrict results to only class place and type city: tag=place:city,\nTo restrict the results to all of OSM class place: tag=place",
+									"description": "Restricts the autocomplete search results to elements of specific OSM class and type.  Now supports multiple tags. Examples :-\n1. To return only cities: tag=place:city\n2. To return only types of place, such as Suburbs, Towns and Cities, use a wildcard: tag=place:*\n3. To restrict results to specific types of place: tag=place:city,place:town,place:village\n4. To restrict results to cafes: tag=amenity:cafe",
+									"disabled": true
+								},
+								{
+									"key": "dedupe",
+									"value": "0",
+									"description": "Sometimes you have several objects in OSM identifying the same place or object in reality. The simplest case is a street being split in many different OSM ways due to different characteristics. Our Geocoder will attempt to detect such duplicates and only return one match; this is controlled by the dedupe parameter which defaults to 1. Since the limit is, for reasons of efficiency, enforced before and not after de-duplicating, it is possible that de-duplicating leaves you with less results than requested.",
```

Comment:
0

Reply:
All the integer value also comes in quote after exporting it from Postman.